docs/serverless-v2/03-03-lambda-processing-phases.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Lambda processing phases
+type: Details
+---
+
+From the moment you create or update a lambda (Function CR) until the time it reaches the final `Running` status phase, lambda goes through three processing phases:
+
+1. `Initializing`
+2. `Building`
+3. `Deploying`
+
+The diagrams illustrate these three core phases of a lambda processing circle that the Function Controller handles. It also lists all custom resources involved in this process and specifies in which cases their update is required.
+
+>**NOTE:** Before you start reading, see the [Function CR](#custom-resource-function) document for its detailed definition, description of all lambda status phases, its substages, and reasons.
+
+## Initializing
+
+This initial phase starts when you create a Function CR with configuration specifying the lambda's setup. It ends with creating a ConfigMap and a TaskRun CR ready for the lambda image built.
+
+Updating an existing lambda involves an image rebuild only if any previous lambda processing phase failed (**phase: Failed**), or if you change lambda's configuration, so its body (**function**) or dependencies (**deps**). An update of lambda's environmental variables doesn't require image rebuild, and it only affects KService in the `Deploying` phase.
+
+> **NOTE:** Each time you update lambda's configuration, the Function Controller deletes all previous TaskRun CRs for the given lambda's **UID** each time you update lambda's configuration.
```

Comment:
```suggestion
> **NOTE:** Each time you update lambda's configuration, the Function Controller deletes all previous TaskRun CRs for the given lambda's **UID**.
```
One `each time..` too many :) You can also remove the first part and leave the last - it's up to you :)